src/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -720,25 +720,10 @@ public void InsertDefault()
         }
 
         [Fact]
-        public void InsertRangeNoOpIdentity()
```

Comment:
I get nervous whenever a code change is associated with removal or changing unit tests. It suggests an observable behavioral breaking change has been made. Why did you remove this?